Service/db/dbUtils.js

```
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+const ENV = process.env;
+const devConnectionString = `mongodb+srv://${ENV.DB_USER}:${ENV.DB_PASS}@cluster0.pa1un.mongodb.net/${ENV.DB_DEV_NAME}?retryWrites=true&w=majority`;
+const prodConnectionString = `mongodb+srv://${ENV.DB_USER}:${ENV.DB_PASS}@cluster0.pa1un.mongodb.net/${ENV.DB_PROD_NAME}?retryWrites=true&w=majority`;
+
+module.exports.connectTest = async (name) => {
+  const testConnectionString = `mongodb+srv://${ENV.DB_USER}:${ENV.DB_PASS}@cluster0.pa1un.mongodb.net/${name}?retryWrites=true&w=majority`;
+  await mongoose.connect(testConnectionString);
+};
+
+module.exports.connectDev = async () => {
+  await mongoose.connect(devConnectionString);
+};
+
+module.exports.connectProd = async () => {
+  await mongoose.connect(prodConnectionString);
+};
+
+module.exports.closeDatabase = async () => {
+  await mongoose.connection.dropDatabase();
+  await mongoose.connection.close();
+};
```

This function is pretty dangerous since it drops the current database so we should probably add some safety feature to prevent accidentally deleting our dev or prod db. 
Here's what I suggest:
- In the `closeDatabase` function, check to make sure the current database name is not the same as the dev/prod databases. (I believe you can get the current db name using `mongoose.connection.name`). If it is the same as the dev/prod db than we should probably throw an error and/or log it as an error.
- Rename the function to `deleteDatabase()` to clarify what it really does.

---

Yeah checking the names is a great idea for safety. I added the changes in the newest commit. The `deleteDatabase()` now checks for the dev and prod db names and throws an error if found. I tested the function and it successfully prevents deleting the prod/dev db's. I also added a `closeDatabase()` to allow closing dev/prod connections.

---

Very nice